distribution/ddf-common/src/main/resources/etc/artemis-backup.xml

```diff
@@ -136,8 +144,8 @@
         <!-- Other config -->
         <security-invalidation-interval>600000</security-invalidation-interval>
 
-        <!--<password-codec>org.codice.ddf.broker.security.EncryptionCodec</password-codec>-->
-        <!--<mask-password>true</mask-password>-->
+        <password-codec>org.codice.ddf.broker.security.EncryptionCodec</password-codec>
```

Comment:
After the hero process, we found out that external clients connecting to the broker using the CORE protocol are be required to use this implementation of the password codec if we include these two lines. External clients should not have to have to know about our implementation of this codec and should find some other means of password encryption such as using SSL connections connections. Will remove these these lines to allow connections to go through using CORE.